paddle/framework/op_registry.h

```diff
@@ -358,30 +357,20 @@ class OpKernelRegistrar : public Registrar {
 /**
  * Macro to register Operator.
  */
-#define REGISTER_OP(op_type, op_class, op_maker_class)                        \
+#define REGISTER_OP(op_type, op_class, op_maker_class, grad_op_type,          \
+                    grad_op_class)                                            \
```

Comment:
Just `USE_OP(OP)` is OK. because we always register forward operator and backward operator in same `.cc/cu` file.

`USE_OP(OP)` will force C++ linker to include all symbols in that `.cc/cu` file including forwarding operator and backwarding operator.